.github/workflows/build.yml

```diff
@@ -92,3 +92,14 @@ jobs:
         env:
           JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           JRELEASER_PROJECT_VERSION: ${{ steps.version.outputs.VERSION }}
+      
+      - name: Configure AWS
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-2
+          role-to-assume: arn:aws:iam::128997144437:role/Development
+      
+      - name: Publish STABLE jars
+        run: ./gradle.sh publish
```

Comment:
for this to work you'll probably need to make sure the `AWS_SESSION_TOKEN` you get from `configure-aws-credentials` is also propagated

Add a similar line here:
https://github.com/seqeralabs/migtool/blob/28a4e1a9e30489a4247b1d224429412c0f168f11/build.gradle#L84

Reply:
oh nevermind [3e42af8](https://github.com/seqeralabs/migtool/pull/33/commits/3e42af8cfc018494fb1727a49d7e3dc394e574ee)